### PR TITLE
BPTL Dashboard: Send email reminders to participants eligible for home collection kit & survey

### PR DESCRIPTION
**Pull request by abhinavjonnada82**
Upon successful kit shipment & kit receipt, email reminders are sent out to participants eligible for home collection kit & survey

This PR addresses following issues:
https://github.com/episphere/connect/issues/802
https://github.com/episphere/connect/issues/803
https://github.com/episphere/connect/issues/804

Related PR: https://github.com/episphere/connectFaas/pull/460

Checklist:
- [X] Code cleanup
- [X] Check for ES Lint warnings
- [ ] Verify test cases
- [x] Check for GIT conflicts 
- [ ] Verified unit tests pass with current changes [N/A]
- [ ] Any dependencies or modules  [N/A]
- [x] Attach PoC
- [ ] Add notes

PoC:
Mouthwash Reminder:
Screenshot 2023-11-15 at 2 10 32 PM

Survey Reminder:
Screenshot 2023-11-15 at 1 45 58 PM

Acknowledgement:
Screenshot 2023-11-16 at 5 52 47 PM

### CODE REVIEW

File: src/pages/homeCollection/kitsReceipt.js [hunk 1]
```diff
@@ -212,6 +211,30 @@ const storePackageReceipt = async (data) => {
       enableCollectionCheckBox();
       document.getElementById("packageCondition").setAttribute("data-selected","[]");
     }
+    
+    const returnedPtInfo = await response.json().then(data => { return data.response })
```
Comment (jhflorey) [inline src/pages/homeCollection/kitsReceipt.js:215]: there seems to be duplicated code with line 115. Should we create a common function to prepare data.

File: src/emailTemplates.js [hunk 2]
```diff
@@ -42,33 +55,58 @@ export const baselineEmailTemplate = (data, isClinical) => {
...
         <br/>
-        Thank you for your commitment to helping us learn how to better prevent cancer.
-        <br/>
+        Thank you for completing your mouthwash home collection kit! We have your sample. Next, please share some information about the day you collected your sample. Visit <a href=${appLocation}>MyConnect app</a> and log in using to complete a short survey. This survey asks questions about the day you collected your sample, so it is important to complete it as soon as you can.
+        Have questions? Please contact the <a href=${supportLocation}>Connect Support Center.</a>
+        ${returnFooterTemplate()}`
+}
+
+export const baselineMWKitRemainderTemplate = (ptName) => {
+    const {appLocation, supportLocation} = getSupportUrls();
```
Comment (Gbarra9) [inline src/emailTemplates.js:74]:
```suggestion
    const { appLocation, supportLocation } = getSupportUrls();
```

small nit: spacing

File: src/pages/homeCollection/kitShipment.js [hunk 3]
```diff
@@ -111,7 +111,28 @@ const setShippedResponse = async (data) => {
...
+      attempt: "1st contact",
+      category: "Biospecimen Home Collection Kit Reminder",
+      token: returnedPtInfo.token,
+      uid: returnedPtInfo.uid,
+      read: false
+    };
+    try {
+      await(sendClientEmail(emailData));
+    }
+    catch (e) {
```
Comment (Gbarra9) [inline src/pages/homeCollection/kitShipment.js:132]:
```suggestion
    } catch (e) {
```
small nit

File: src/pages/homeCollection/kitsReceipt.js [hunk 4]
```diff
@@ -212,6 +211,30 @@ const storePackageReceipt = async (data) => {
...
+
+    if(returnedPtInfo.surveyStatus !== conceptIds.modules.submitted) {
```
Comment (Gbarra9) [inline src/pages/homeCollection/kitsReceipt.js:217]:
```suggestion
    if (returnedPtInfo.surveyStatus !== conceptIds.modules.submitted) {
```

File: src/pages/homeCollection/kitsReceipt.js [hunk 5]
```diff
@@ -212,6 +211,30 @@ const storePackageReceipt = async (data) => {
...
+        attempt: "1st contact",
+        category: "Biospecimen Home Collection Survey Reminder",
+        token: returnedPtInfo.token,
+        uid: returnedPtInfo.uid,
+        read: false
+      };
+      try {
+        await(sendClientEmail(emailData));
+      }
+      catch (e) {
```
Comment (Gbarra9) [inline src/pages/homeCollection/kitsReceipt.js:233]:
```suggestion
      } catch (e) {
```

**Review by Gbarra9 — APPROVED** [review]

**Review by ramavavi — APPROVED** [review]

**Comment by ramavavi** [comment]
Looks good for me.

**Review by jhflorey — APPROVED** [review]

**Review by Gbarra9 — APPROVED** [review]
Looks good.

**Review by JoeArmani — APPROVED** [review]
looks good

**Pull request merged**